### DISCUSSION TITLE
GitHub should not turn into a social media platform?

### DISCUSSION THREAD

**Discussion by Nllii** [community/community#18203] [Other Feature Feedback, Questions, & Ideas]
GitHub should not turn into a social media platform? 

I have seen this before on a flash site back in 2005, where the community created a lot of flash projects(swf) and shared templates for sites, shared codes  and cool hacks(mostly flash runtime injection vulnerability to win flash games) and ideas on flash security;  then the site admin added social media into it. This  turned the site into spam, political and propaganda playground.

**Comment by schmetti-dev** [comment]
I'm not a native English speaker and I'm confused by questions with a not

Does it now say YES, I agree with the questioner and I am AGAINST having any connotation of social media
or does it mean NO, I am against having any connotation of social media?

Sorry, but 🤷‍♂️

**Reply by was07** [reply]
It says "do you agree" in the question. yes would mean you don't want Github to be a social media.

**Reply by schmetti-dev** [reply]
Ok thanks 🙈

**Reply by librity** [reply]
It is awkwardly worded

**Reply by ghost** [reply]
It sounds like I wrote it. Bet the person might be from the south. We put not in the oddest places.

**Comment by DhanushNehru** [comment]
The github user should have an option whether to view the github as a social media platform or the regular github profile. He can opt in or opt out based on his preferences.

**Comment by Zerotask** [comment]
It depends on your definition. Personally I like to discuss, share thoughts and get to know people with similar interests.

**Reply by WebCrew** [reply]
I think the same - its a great way to find other people with the same interests maybe even for a future project.

**Reply by babekasadli** [reply]
agree with u

**Reply by librity** [reply]
I like that too, I don't see how GitHub achievements make that any better.

**Comment by wallaceleonel** [comment]
Acredito que a visualização e exposição do trabalho tem que ser pessoal, as pessoas podem optar se querem seu perfil visível ou não .

**Comment by epistax1s** [comment]
Conservatives... How will the elements of a social network prevent you from working with open source?

**Comment by ERPedersen** [comment]
GitHub is both a code sharing and publishing service, and a social networking site for programmers. One of the features that makes it a social networking site is the feature you used to create this discussion. How do you want to promote collaboration without social interaction?

**Reply by Cactusinhand** [reply]
If there is social, there will be too social. 

This has been proved by all social media, like Facebook, Twitter etc.
But apparently, Github is not a social paltform where you getting in touch with your networking friends or sharing your daily life. It is all about code, projects, idea.

Please let social be social, let Github be Github.

**Comment by Alexanderdivv** [comment]
it just specific for sharing code, in my opinion of course

**Comment by hudy0** [comment]
Yes I surely agree as you know GitHub number one  sharing CODE and publishing service.

**Comment by mcspach** [comment]
I completely agree. There is NO reason to gamify a platform like this. If anyone has a reason that gamification and social-media aspects would improve the services that Github offers, I would love to have my opinion challenged.

**Comment by keybraker** [comment]
Github should stay as is, and make it abundantly clear where the social aspect (event, etc) of it lies. Maybe create a social button you click in order to engage with it.

**Comment by Rumata-SL** [comment]
GitHub is a social network.

**Comment by FranzRome** [comment]
In my opinion **GitHub is already some sort of social network**. It's just not like the others _(luckily)_ but there are still socialization elements on this platform. So, **currently GitHub is ok**, but if they wolud add videos memes post or any other thing not related to code it would be the ruin of this beautiful echosystem.

**Comment by WebCrew** [comment]
Come on folks - social network or not, my goodness, **the whole world has become a fucking social network**. I think it's not bad to pass the time here on GitHub with discussions and nicer user accounts - if you don't want that, **you can use the boring console** :) As if there were really nothing more important than nagging about the fact that GitHub now has something more of an SN. I don't think it's bad, on the contrary I think it's good as long as it doesn't mutate into a lousy Facebook (I hate FB) :)

**Reply by librity** [reply]
> as long as it doesn't mutate into a lousy Facebook (I hate FB) :)

Don't worry, it will 😉
